docs/book/src/developers/releasing.md

```diff
@@ -88,6 +88,13 @@ Example versions:
     - Move items out of "Uncategorized" into an appropriate section.
     - Change anything attributed to "k8s-cherrypick-robot" to credit the original author.
     - Fix any typos or other errors.
+    - Add a "Details" section with a link to the full diff:
```

Comment:
I've been adding this section by hand because I've found the full diff very useful. (A lot of commits have release note NONE and don't show up here.) So I thought I'd add it to the release instructions. LMK if this is useful.